Makefile

```diff
@@ -40,6 +40,7 @@ install_c:
 ### Protobuf
 
 protoc_all: protoc_libs protoc_merkle protoc_abci protoc_grpc protoc_proto3types
+	@ls **/**/**.pb.go | xargs -n1 -IX bash -c 'sed s/,omitempty// X > X.tmp && mv X{.tmp,}'
```

Comment:
I'd rather leave omitempty property in pb.go files. It reduces the size of the end data when sent over p2p network. I'd prefer to use custom marshallers with EmitDefaults set to true for RPC specifically.